### PR TITLE
Second project code review

### PR DESCRIPTION
**Pull request by pastorp3**
our project has the next requirements:

- Add linterns file 

- HTML and CSS structure

-designed as the original page

- We use only HTML and css

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Let's see...

- Great job adding the linter config files. But you have a typo in your folder name. Right now its `.githubworkflows` but it should be `.github/workflows`. The `workflows` folder should be inside `.github` folder. The linter config file should be inside the `workflows` folder. Here are the instructions: https://github.com/microverseinc/linters-config/tree/master/html-css

- Files not related to the project should not be committed. Please remove this `.rar` file from your repository.

image

- There are a lot of errors and warnings in your HTML file. Please visit https://validator.w3.org/#validate_by_input, paste your HTML code, go through each of the errors and warnings and fix them

image

- You have one invalid CSS property. Check the comment.

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
+/* body */
+
+body {
+    background-color: #eceef1;
+    float: center;
```
Comment (tirthajyoti-ghosh) [inline css/main.css]: `float` can only have `left` or `right` as its value. `center` is not a valid `float` value.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
- Remember to fix the typo in your github workflows folder name. Otherwise, github actions won't work.
- Be sure to remove any irrelevant files from your repository.
- Please make sure to fix all the errors and warnings in your HTML code.


Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻

**Comment by pastorp3** [comment]
- We change .github folder and linters now it's working

- We delete the not related file

- We change css style in body

- We already fix all of the error in HTML validation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                </ul>
+            </li>
+            <li class="list-element"><input type="button" value="Sign in" id="sign-bttn">
+            </li>
+        </ul>
+    </nav>
+    <br>
+<!-- End nav bar-->
+
+<!-- Start main section-->
```
Comment (voscarmv) [inline index.html:23]: Rather than only indicating the main section with an HTML comment, please use a proper `<main>` semantic tag here.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far, team. 👍 
- Well-structured `README` file.
- Good, descriptive, Pull Request message.

## However, there are some changes required 👀 🧰 🔨 🔧 
- The top right corner button is overflowing to the right side of the creen. Even when zooming in and out, the button still overflows to the right. Kindly fix this.
image
- The product names are only images. The products below the INTUIT logo are supposed to be composed of the logo followed by text, not just an image. Kindly fix this as well.
image
- Also, if possible, fix the INTUIT logo so that it does not look stretched. It should look similar to the original:
image
- Use the `<main>` semantic tag to denote the main section of your `index.html`.

**[OPTIONAL]**
- Please improve your commit messages in the future. Messages such as `edit html` or `html design` say very little about what was done. Make a short summary of the specific parts of the project that you worked on, rather than generic descriptions.

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by pastorp3** [comment]
- icon change to icon and text

- add main tag to HTML

-gray log of intuit

- fix nav bar

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

### Great work team. The page looks quite good and you implemented all that was requested by my colleagues above.  :clap: 

If you want to make the projects even better, please consider some optional changes below:
- [optional] Use the same background color as the original.
- [optional] Use less id's for the next projects because they have to be unique and that is hard to manage.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by janis-jenny** [comment]
Second project code review:

This is our last changes in the second project before I merge it:

-Icons changed to icons and text
-Changed color of Intuit logo to grey
-Added more margin to the form 
-Changed the navbar to align its items and to have the same font
-Added main tag to HTML.

**Pull request merged**